.github/workflows/continuous-integration.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-composer-
 
-      - run: composer install --no-interaction
+      - run: composer install --no-interaction --no-plugins
```

Comment:
In [docker-compose.yml](https://github.com/pmattmann/ecamp3/blob/3cfad3e6a9876093b7582fb2972b215ea300f533/docker-compose.yml#L53) the following options are enabled:
- --no-interaction 
- --no-plugins 
- --no-scripts 
- --prefer-dist 
- --no-suggest

In the case of CI, shouldn't we add all of them as well?

Reply:
Yes, agreed

Reply:
done